opennlp-dl/pom.xml

```diff
@@ -42,6 +42,11 @@
       <artifactId>onnxruntime_gpu</artifactId>
       <version>${onnxruntime.version}</version>
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
```

Comment:
We could use https://github.com/netplex/json-smart-v2 for parsing the json input - it is also ASLv2. It is just 120KB instead of 2MB from Jackson (+ Jackson Core).

Reply:
Alternative for JSON-P would be https://johnzon.apache.org/ - That being said, I am also ok with using `jackson-databind` as we aren't in the core project ;-)